src/components/modules/epics/ContributionsEpicButtons.tsx

```diff
@@ -79,50 +85,75 @@ const SecondaryCtaButton = ({
     );
 };
 
+interface ContributionsEpicButtonsProps {
+    variant: Variant;
+    tracking: EpicTracking;
+    countryCode?: string;
+    onOpenReminderClick: () => void;
+    submitComponentEvent?: (event: OphanComponentEvent) => void;
+    isReminderActive: boolean;
+}
+
 export const ContributionsEpicButtons = ({
     variant,
     tracking,
     countryCode,
     onOpenReminderClick,
-}: {
-    variant: Variant;
-    tracking: EpicTracking;
-    countryCode?: string;
-    // eslint-disable-next-line @typescript-eslint/ban-types
-    onOpenReminderClick: Function;
-}): JSX.Element | null => {
+    submitComponentEvent,
+    isReminderActive,
+}: ContributionsEpicButtonsProps): JSX.Element | null => {
+    const [hasBeenSeen, setNode] = useHasBeenSeen({}, true);
+
     const { cta, secondaryCta, showReminderFields } = variant;
-    const showReminderCta = !getCookie('gu_epic_contribution_reminder');
+    const hasSetReminder = getCookie('gu_epic_contribution_reminder');
```

Comment:
I renamed this because I thought it was slightly confusing to be called `showReminderCta` but the condition to show the reminder CTA is `showReminderCta && showReminderFields`